commit aef033683d4cfeee67b68faad4e78517592c7d8e
Author: Francesc Puig <f.puig.i.plana@accenture.com>
Date:   Mon Oct 23 16:50:23 2023 +0200

    use col.default

diff --git a/ddlparse/__init__.py b/ddlparse/__init__.py
index 843a9fe..a8ecc0f 100644
--- a/ddlparse/__init__.py
+++ b/ddlparse/__init__.py
@@ -8,7 +8,7 @@
 from .ddlparse import *

 __copyright__    = 'Copyright (C) 2018-2021 Shinichi Takii'
-__version__      = '1.10.0'
+__version__      = '1.11.0'
 __license__      = 'BSD-3-Clause'
 __author__       = 'Shinichi Takii'
 __author_email__ = 'shinichi.takii@shaketh.com'
diff --git a/ddlparse/ddlparse.py b/ddlparse/ddlparse.py
index 823a342..8cca290 100644
--- a/ddlparse/ddlparse.py
+++ b/ddlparse/ddlparse.py
@@ -512,7 +512,7 @@ class DdlParseTable(DdlParseTableColumnBase):

         return self._columns.to_bigquery_fields(name_case)

-    def to_bigquery_ddl(self, name_case=DdlParseBase.NAME_CASE.original, project="project", use_length=False, use_default=False):
+    def to_bigquery_ddl(self, name_case=DdlParseBase.NAME_CASE.original, project="project", use_length=False):
         """
         Generate BigQuery CREATE TABLE statements

@@ -541,15 +541,13 @@ class DdlParseTable(DdlParseTableColumnBase):
                 # no array data type
                 type = col.bigquery_standard_data_type
                 not_null = " NOT NULL" if col.not_null else ""
+                default_column = ""
+                if col.default:
+                    default_column = " DEFAULT " + f"'{col.default}'" if type != 'NUMERIC' else f"{col.default}"
                 length = ""
                 if use_length:
                     length = f"({col.length})" if (col.length is not None and str(col.length).isdigit()) else ""

-                default_column = ""
-                if use_default:
-                    default_value = f"'{col.default}'" if col.bigquery_standard_data_type != 'NUMERIC' else f"{col.default}"
-                    default_column = f" DEFAULT {default_value}" if col.default is not None else ""
-
             else:
                 # one or multiple dimensional array data type
                 type_front = "ARRAY<"
